ui/src/sources/containers/OnboardingWizard.tsx

```diff
@@ -141,6 +155,11 @@ class OnboardingWizard extends PureComponent<Props, State> {
     )
   }
 
+  // WelcomeStep
+  private isWelcomeComplete = () => {
+    return true
```

Comment:
should this always return true?

Reply:
Yes, this first step is always complete.